tracer/src/Datadog.Trace/TraceContext.cs

```diff
@@ -193,25 +192,21 @@ private void PropagateMetadata(ArraySegment<Span> spans)
             }
         }
 
-        /// <summary>
-        /// When receiving chunks of spans, the backend checks whether the aas.resource.id tag is present on any of the
-        /// span to decide which metric to emit (datadog.apm.host.instance or datadog.apm.azure_resource_instance one).
-        /// </summary>
-        private void DecorateWithAASMetadata(Span span)
+        private void AddAASMetadataToTraceChunk(Span span)
```

Comment:
We should rename the method since only adds tags to the specified span, not to a trace chunk. We could rename to `AddAASMetadataToSpan()`, but that seems redundant to me since the parameters make it clear, so I suggest:
```suggestion
        private void AddAASMetadata(Span span)
```

Reply:
You'll need to update the call to this method, too, which was not included in my suggestion above :)